docs/tasksDir/mutating-policies.md

```diff
@@ -11,22 +11,61 @@ Mutating policies receive an object request and rebuild this incoming object
 of the policy. The request will proceed through the Kubernetes API, potentially being
 evaluated by other policies.
 
+If you want to allow the behavior of mutating requests,
+set the `ClusterAdmissionPolicy.mutating` field to `true`.
+
+However, if you set the `ClusterAdmissionPolicy.mutating` field to `false`,
+the mutated requests will be rejected.
+
+# Why mutating policies can be dangerous
+
+### Unreviewed mutating policies can introduce vulnerabilities
+
 :::danger
 To prevent system abuse, Kubewarden administrators should review mutating
 policies: mutating policies could for example modify a workload, such that it
 allows for privileged container creation.
+:::
+
+#### Solution
 
 If in doubt, split policies into mutating, and validating policies, instead of
 writing or deploying policies that both validate and mutate. This is particularly
 important when using a DSL (such as Rego) to build complex policies.
-:::
 
-If you want to allow the behavior of mutating requests,
-set the `ClusterAdmissionPolicy.mutating` field to `true`.
+### Misconfigured mutating policies together with 3rd party Kubernetes Controllers can get stuck in an infinite loop
 
-However, if you set the `ClusterAdmissionPolicy.mutating` field to `false`,
-the mutated requests will be rejected.
-For example, create the following `ClusterAdmissionPolicy` with the `mutating` field set to `true`:
+:::danger
+Mutating policies return requests that proceed through the Kubernetes API. If
+there are other Kubernetes Controllers that listen for those same resources,
+they may mutate them back in a follow-up request. This could lead to an
+infinite feedback loop of mutations.
+:::
+
+#### Solution
+
+Perform the mutating against the lower type of resource (e.g: Pod) or the
+highest (e.g: Deployment). Each has its trade-offs:
+
+1. Have the policy inspect low level resources (e.g: Pod). Different kind of
+   Kubernetes resources (be them native or CRDs) can create Pods. By having the
+   policy target Pod objects, there's the guarantee all the Pods are going to be
+   compliant.
+   However, this could lead to confusion among end users of the cluster: their
+   high level Kubernetes resources would be successfully created, but they would
+   stay in a non reconciled state. For example, a Deployment creating a
+   non-compliant Pod would be created, but it would never have all its replicas
+   running. The end user would have to do some debugging to finally understand why
+   this is happening.
```

Comment:
This should be dropped. That applies to validating policies, not to mutating ones.